compendium/DeclarativeServices/test/TestComponentLifecycle.cpp

```diff
@@ -252,4 +278,75 @@ TEST_F(tServiceComponent, testDelayedComponent_LifeCycle) //DS_TOI_52
   auto sRef2 = ctxt.GetServiceReference<test::Interface2>();
   EXPECT_FALSE(static_cast<bool>(sRef2)) << "Service must not be available after it's dependency is removed";
 }
+
+//Function to validate lazy loading of delayed component
+bool isBundleLoaded(const std::string bundleName)
```

Comment:
`bundleName` should be passed in as `const std::string&` or `std::string`, if you want to allow move semantics.

this entire function has general use and could be used by other tests. I suggest moving it into the TestUtils.cpp file and renaming the function to a more descriptive name, something like `isBundleLoadedInThisProcess`